client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -122,11 +123,12 @@ export const PatternLibrary = ( {
 
 	const recordClickEvent = (
 		tracksEventName: string,
-		view: PatternView,
-		typeFilter: PatternTypeFilter
+		view?: PatternView,
+		typeFilter?: PatternTypeFilter
 	) => {
 		recordTracksEvent( tracksEventName, {
 			category,
+			search_term: searchTerm ? searchTerm : undefined,
```

Comment:
```suggestion
			search_term: searchTerm || undefined,
```
Nitpicking, but we might as well do this.